src/pages/ManageUser/personalDetails.tsx

```diff
@@ -1,3 +1,17 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
why are you disabling  typescript any check?

Reply:
any is need as the error type in the error async catch